jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -197,49 +125,100 @@ def ConvertAndCompare(self, func_jax: Callable, *args,
     match the value from the JAX execution.
 
     Args:
-      custom_assert: a function that will be called
-        `custom_assert(result_jax, result_tf)` to assert equality of the
-        results. Use this function when JAX and TF produce different results.
-        This function is only used for "eager" and "graph" modes by default, not
-        for the "compiled" mode, because in that case we expect the results to
+      func_jax: the function to invoke (``func_jax(**args)``)
+      args: the arguments.
+      enable_xla: if True, allows the use of XLA ops in jax2tf.convert
+        (default: True).
```

Comment:
nit: "XLA ops" -> "TensorFlow XLA ops" or "TFXLA ops" to be more explicit and consistent with [the documentation](https://github.com/google/jax/tree/master/jax/experimental/jax2tf#tensorflow-xla-ops)